readme.md

```diff
@@ -82,7 +100,15 @@ var Panda = function(name, age) {
 
 Your Answer:
 ```text
-
+eat_bamboo: function() {
+  var self = this;
```

Comment:
`self` is meant to be assigned outside of a function, so that you have access to the outer context when/if the `this` inside the function has it's own context (e.g. a function that acts as a callback to an event listener has the element in `this`.